content/en/docs/concepts/overview/components.md

```diff
@@ -23,14 +23,45 @@ Here's the diagram of a Kubernetes cluster with all the components tied together
 {{% /capture %}}
 
 {{% capture body %}}
-## Control Plane Components
+## Node components
 
-The Control Plane's components make global decisions about the cluster (for example, scheduling), as well as detecting and responding to cluster events (for example, starting up a new {{< glossary_tooltip text="pod" term_id="pod">}} when a deployment's `replicas` field is unsatisfied).
+In Kubernetes, your application runs inside a group of containers called
+a {{< glossary_tooltip text="_pod_" term_id="pod" >}}. The idea of a pod
+is to isolate different workloads from each other. The computers
+that run pods are called _nodes_, and one node can run pods from many
+different workloads.
+
+To make this happen, Kubernetes runs several components on each node:
+
+### kubelet
+
+{{< glossary_definition term_id="kubelet" length="all" >}}
+
+### kube-proxy
+
+{{< glossary_definition term_id="kube-proxy" length="all" >}}
+
+### Container runtime
+
+{{< glossary_definition term_id="container-runtime" length="all" >}}
+
+## Control plane components
+
+The more different kinds of pod you want to run in your cluster, the
+more it helps to have systems to manage and share system resources,
+to make sure pods can communicate with other pods they need to talk to,
+and to make sure that the cluster is operating securely.
+
+The Control Plane's components make global decisions for your cluster
+(for example, scheduling), as well as detecting and responding to workload
+level events (for example, starting up a new {{< glossary_tooltip text="pod"
+term_id="pod">}} when a deployment's `replicas` field is unsatisfied).
```

Comment:
Could also link to glossary for deployment

Reply:
Because this is the overview, I'm wary of throwing lots of links at the reader. I'm also open to being persuaded.